src/pages/homePage/HomePage.tsx

```diff
@@ -5,16 +5,36 @@ import AboutUs from "./components/AboutUs/AboutUs";
 import IntroSearch from "./components/IntroSearch/IntroSearch";
 import Carusel from "./components/Carusel/Carusel";
 import ChefOfTheWeek from "./components/ChefOfTheWeek/ChefOfTheWeek";
-import { dishes } from "../../assets/mockdata/signature";
+import { useEffect, useState } from "react";
+import { fetchDishes, transformToCardItem } from "./apiHomePage";
+import { Dish } from "../../shared/interfaces/Dish";
+import CardItem, { CardItemList } from "../../shared/interfaces/CardItem";
 
 const Home = () => {
+    const [loadedDishes, setloadedDishes] = useState<CardItem[]>([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        const getDishes = async () => {
+            try {
+                setIsLoading(true);
```

Comment:
I see that you set this parameter to true but is never being reseted to false (after there is an error or a success response)